### PR TITLE
Allow to disable node features

### PR DESCRIPTION
**Pull request by TrySound**
My use case is esm bundle with conventional `NODE_ENV` which should not
be transpiled but there's also `global` usage which should be ready to
use in browsers.

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
sorry, just got back from vacation, will try to take a look

File: readme.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Plugin to insert node globals including so code that works with browserify shoul
 Plus `process.nextTick` and `process.browser` are optimized to only pull in
 themselves and __dirname and __filename point to the file on disk
 
-Only option beyond the default plugin ones is an optional basedir which is used for resolving __dirname and __filename.
+Only option beyond the default plugin ones is an optional `baseDir` which is used for resolving __dirname and __filename.
```
Comment (calvinmetcalf) [inline readme.md]: I think there are more changes then just this

Comment (TrySound) [reply readme.md]: Done

**Comment by TrySound** [comment]
Friendly ping @calvinmetcalf

**Comment by calvinmetcalf** [comment]
woops, will try to look at this today

**Comment by calvinmetcalf** [comment]
v1.3.0

**Pull request merged**